File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashMap<Integer,Integer> hashMap = new HashMap<Integer,Integer>();
        for(Integer num : nums){
            hashMap.put(num,hashMap.getOrDefault(num,0)+1);
        }
        for (Integer key : hashMap.keySet()){
            Integer value = hashMap.get(key);
            if(value>1){
                return true;
            }
        }
        return false;
        
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hashmap = {}
        result = []
        for num in nums:
            if num not in hashmap:
                hashmap[num] = 1
            else:
                hashmap[num] += 1
        sorted_hashmap = dict(sorted(hashmap.items(), key=lambda item: item[1], reverse=True))
        sorted_keys = list(sorted_hashmap.keys())
        return sorted_keys[:k]
        